homeassistant/components/cloud/prefs.py

```diff
@@ -305,7 +306,8 @@ async def _save_prefs(self, prefs):
             self._hass.async_create_task(async_create_catching_coro(listener(self)))
 
     @callback
-    def _empty_config(self, username):
+    @staticmethod
+    def _empty_config(username):
```

Comment:
Since `self._hass` is no longer required for the webhook_id, pylint was complaining about `method` versus `function`.
I felt it was best to set it as a static method, rather than introduce a pylint exception.

Reply:
I think here staticmethod is suited - but happy to change if someone disagrees